docs/faq/templates.md

```diff
@@ -54,8 +54,8 @@
 
 ??? warning "I'm not able to run all templates!"
 
-	Nuclei follows the default list of templates that are excluded from default run and are listed [here](https://github.com/projectdiscovery/nuclei-templates/blob/master/.nuclei-ignore), **This is to ensure that no one runs any templates that are not meant to run along with all templates.** We also want to make sure that no one is excluded from running safe scans. We do not suggest that you run them unless you are running them for a specific use.
+	Nuclei respects the default list of templates that are excluded from default run. These are listed [here](https://github.com/projectdiscovery/nuclei-templates/blob/master/.nuclei-ignore), **This is to ensure that some templates are not run along with all templates.** We also want to make sure that no accidentally runs unsafe scans. We suggest only running them for a specific use in a scan.
```

Comment:
"are not run along with all templates" could be rewritten with "are not executed by default"